src/components/plots/linePlot.ts

```diff
@@ -4,6 +4,8 @@ module Plottable {
 export module Plots {
   export class Line<X> extends XYPlot<X, number> {
 
+    public static SELECTION_CLASS = "line";
```

Comment:
I don't think this is the right name for the field. People might grab the elements without using d3, for example.